### ISSUE TITLE
[Usable – Make Code – Home page] Visually appearing ‘View all’ Button looks like normal text on ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3512]
**User Experience**:
User rely on keyboard will find difficulty to interact with the tooltip information if  tooltip is not accessible through keyboard as user won't be able to perform functionality over such control.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify the ‘View all’ button by using developer tool.

**Actual Result:**
Visually appearing ‘View all’ Button looks like normal text on ‘Home page’, due to this users will not be able to see the difference b/w text and button and will face difficulties while navigating on the page.
 
**Expected Result:**
Boundary lines should be define for ‘View all’ button under ‘Home page’, so that users can see the difference b/w text and button and will not face any difficulties while navigating on the page.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#A11yUsable, #A11ySev4, #Accessibility, #Benchmark, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

![Usable_View All button looks as a normal text](https://user-images.githubusercontent.com/72871089/97540441-a6b6bc00-19e9-11eb-8d7a-c998b9ad72bb.jpg)

**Comment by abchatra** [comment]
This behaves like a link and looks like a link. This is how it is designed for all usres.
